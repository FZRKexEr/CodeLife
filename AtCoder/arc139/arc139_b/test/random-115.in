100
90 9 14 4 10 7
42 4 10 17 17 3
61 1 5 18 16 11
95 16 17 2 3 4
41 3 12 8 9 5
50 14 4 12 10 18
51 1 11 10 15 10
94 12 16 4 12 2
57 8 17 8 17 10
48 3 9 19 7 19
59 18 20 6 14 17
72 2 1 15 5 1
37 5 3 3 17 10
19 9 13 6 15 17
47 13 9 16 16 16
68 2 1 19 13 1
11 20 12 14 2 7
40 14 19 8 2 13
76 6 3 15 12 19
95 1 10 15 9 8
12 8 17 2 19 4
99 3 9 10 4 17
35 15 13 20 11 19
5 4 5 6 5 9
97 15 10 1 2 11
18 12 4 10 5 2
80 18 11 4 10 5
51 4 4 8 3 10
12 9 16 17 9 16
55 3 9 19 16 7
47 9 6 3 10 1
24 8 9 14 4 7
81 8 15 2 19 9
37 19 2 8 13 16
74 4 4 20 11 5
89 4 5 13 11 15
20 5 1 2 11 18
99 2 3 6 20 19
90 6 9 19 6 5
15 6 5 2 4 16
1 3 5 20 5 4
77 7 10 7 18 18
38 9 19 13 20 3
28 16 19 1 1 20
93 17 7 12 10 8
24 20 7 3 19 11
91 12 20 15 1 7
14 3 3 19 1 7
76 6 5 11 6 8
59 6 7 2 18 10
27 16 11 6 20 14
86 1 13 20 6 12
62 8 6 13 3 8
15 20 1 4 4 17
57 3 8 13 1 1
38 2 5 11 6 20
88 8 19 17 4 12
37 15 9 15 19 20
66 18 11 5 1 3
42 11 7 8 10 10
34 5 16 18 5 18
91 7 11 4 19 15
30 16 15 17 5 8
11 8 4 5 7 2
96 8 9 17 15 4
96 12 6 17 19 1
88 13 1 5 16 13
52 7 2 3 19 8
29 3 17 15 13 9
32 7 3 5 3 4
57 19 14 17 9 19
75 11 18 18 17 17
53 14 2 2 13 15
6 4 2 11 5 1
39 3 12 13 8 17
60 19 12 19 7 5
44 6 1 3 18 16
45 15 15 9 17 7
29 19 17 20 9 4
90 11 1 8 19 13
6 7 16 2 9 7
74 15 13 4 16 15
48 17 12 2 4 15
50 17 13 6 7 16
15 3 5 8 3 6
13 7 15 20 4 15
33 6 13 11 6 10
16 17 15 18 10 2
66 5 11 6 6 15
79 15 19 10 17 1
88 3 18 17 6 16
56 17 20 5 7 18
34 17 14 18 7 3
1 19 1 17 20 9
48 2 1 14 4 8
31 9 6 18 10 2
6 1 9 14 19 3
68 10 6 16 19 10
61 8 18 20 4 11
81 3 15 19 8 2
