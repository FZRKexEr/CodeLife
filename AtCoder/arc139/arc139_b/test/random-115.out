63
46
148
40
40
66
110
30
78
102
54
72
111
36
112
68
154
42
178
117
24
132
122
9
19
27
40
60
60
61
22
12
57
157
130
246
40
594
50
14
20
144
6
13
70
31
63
42
82
86
34
120
29
60
9
114
44
95
11
56
72
132
16
19
77
16
146
142
61
21
27
122
61
3
59
25
132
21
209
168
12
83
27
48
15
124
37
20
81
34
111
42
14
9
96
28
84
118
54
26
